.github/ISSUE_TEMPLATE/new_component.yaml

```diff
@@ -26,8 +26,9 @@ body:
       label: Is this a vendor-specific component?
       description: A vendor-specific component directly interfaces with a vendor-specific API and is expected to be maintained by a representative of the same vendor.
       options:
-        - label: This is a vendor-specific component
-        - label: If this is a vendor-specific component, I am proposing to contribute this as a representative of the vendor.
+        - label: This is a vendor-specific component.
+        - label: (If a vendor-specific component) I am proposing to contribute and maintain this as a representative of the vendor.
+        - label: (If a vendor-specific component) I am a member of the OpenTelemetry project.
```

Comment:
One downside of requiring membership before proposing a component is that the proposer can no longer use the contributions from creating the component as the means to membership.  This provided a clear path toward membership.

A benefit of this change, though, is that is should give us more confidence going into the proposal that the user is going to maintain the component.  Personally, I think Contrib is large enough at this point that more rigorous rules and expectations are necessary to ensure we can grow any further.